src/util/vocabularyChecker/detectWrongSyntax.ts

```diff
@@ -15,10 +15,14 @@ export const anyWord = new RegExp(/\b\w+\b/g);
 const allowedWords = json;
 export interface Position { fromIndex: number; toIndex: number; }
 
-export function collectInvalidWordsInIssues(text: string) {
+/**
+ * Adds an Issue for every invalid word.
+ * @param text the userinput
+ */
+export function collectInvalidWordsInIssues(text: string): void {
 	const invalidWords = getInvalidWords(text);
 	for (const word of invalidWords) {
-		if (!(allowedWords.includes(word.toLowerCase()))) {
+		if ((!(allowedWords.includes(word.toLowerCase())))) {
```

Comment:
Is this extra parenthesis necessary for this to work? Otherwise I think the extra paranthesis makes it less readable.